tests/vault_action_tests_base.py

```diff
@@ -0,0 +1,77 @@
+from st2tests.base import BaseActionTestCase
+
+from tests.utils import get_config_file_path
+from tests.utils.hvac_integration_test_case import HvacIntegrationTestCase
+
+
+class VaultActionTestCase(HvacIntegrationTestCase, BaseActionTestCase):
+    dummy_pack_config = None
+    secret_v1 = False
+    secret_v2 = False
+    default_token_lease = "1h"
+
+    # in setUp() and tearDown(), explicitly handle super()-like calls.
+    #  - HvacIntegrationTestCase does not call super().
+    #  - BaseActionTestCase does call super().
+
+    def setUp(self):
+        HvacIntegrationTestCase.setUp(self)
+        # self.client = create_client(token=self.manager.root_token)
+        # also mocks hvac.utils.warnings which we might not want.
+        BaseActionTestCase.setUp(self)
```

Comment:
```suggestion
        # Calls BaseActionTestCase.setUp(self)
        super(HvacIntegrationTestCase, self).setUp()
```

Reply:
Huh. Calling super on the class that doesn't call it is a fascinating way to fix the issue. Cool.